### ISSUE TITLE
Typographic Scale Based on Golden Ratio (based on your articles)

### ISSUE THREAD

**Issue by newdesignideas** [HugoGiraudel/ama#78]
Hi Hugo,

I did a small snippet based on one of your articles that may be useful to turned into a Sass component and added as an optional add-on to a blueprint or a framework that enables one to use a typographic scale based on Golden Ratio in a CSS project with the addition of minimum and maximum font sizes.

If you care to have a look. It's here: https://github.com/Veebidisainer/golden-ratio-typographic-scale 

The idea is to add a range where in any font looks good enough and let the Golden ratio do the rest. 

(still a work in progress)

**Comment by HugoGiraudel** [comment]
Hey there,

That’s nice work! I don’t use Sass as much as I used to, so I can’t really comment on how useful it is in a real project, but here are a few things I noticed while looking at the code:
- You use 3 naming conventions for your variables: hyphenated, snakerized and camel-cased. I recommend to stick to one (my love goes to hyphenated).
- You appear to be using variable interpolation a bit arbitrarily. It’s not needed most of the time, can have side effects and hurt readability. I recommend to only use it when necessary.
- I’m usually not a fan of brutally stripping units (with the Sass `strip-unit(..)` function you borrowed from CSS-Tricks, which I happened to write :D) as it’s not a natural way to handle units in maths. This article might help understand my point: https://www.sitepoint.com/understanding-sass-units/.

In any case, that’s seems like a solid idea. Keep up! :)
